sentences/da/_common.yaml

```
@@ -33,6 +39,60 @@ lists:
         out: purple
       - in: brun
         out: brown
+      - in: lyserød
+        out: pink
+      - in: turkis
+        out: turquoise
+  cover_classes:
+    values:
+      - in: markise[rne]
+        out: awning
+      - in: persienne[rne]
+        out: blind
+      - in: gardin[erne]
+        out: curtain
+      - in: dør[ene]
+        out: door
+      - in: garageport[ene]
+        out: garage
+      - in: låge[rne]
+        out: gate
+      - in: skærm[ene]
```

This is closer to "persienne" or "rullegardin" in Danish.

```suggestion
      - in: (persienne[(en|r|rne)]|rullegardin[(et|er|erne)])
```